CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Use getByteTimeDomainData to support iOS Safari in meeting demo
+- Updated README to remove use case numbering & incorporate feedback.
```

Comment:
Use present tense, write out "&" as "and", drop period at end to be consistent with other change logs:

e.g.

`Update README to remove use case numbering and incorporate documentation feedback`

Reply:
Updated.